packages/formstr-app/src/components/v1/Index.tsx

```diff
@@ -0,0 +1,108 @@
+import { useMemo, useState } from "react";
+import QuestionCard from "./QuestionCard/QuestionCard";
+import { Button, Dropdown, Menu, MenuProps } from "antd";
+import { AnswerTypes, Field } from "@formstr/sdk/dist/interfaces";
+import { MenuFoldOutlined, MenuUnfoldOutlined } from "@ant-design/icons";
+import { AnswerTypeLabels } from "./constants";
+import { makeTag } from "../../utils/utility";
+import { createForm } from "@formstr/sdk";
+
+export interface IQuestion extends Field {
+  tempId: string;
+  inputSettings: {};
+}
+
+const initialQuestion: IQuestion = {
+  tempId: makeTag(6),
+  question: "Click here to edit",
+  answerType: AnswerTypes.shortText,
+  inputSettings: {},
+};
+
+const getItems = () => {
+  return Object.keys(AnswerTypes).map((answerType: string) => {
+    return {
+      key: answerType,
+      label: AnswerTypes[answerType as AnswerTypes],
+    };
+  });
+};
+
+export const QuestionsList = () => {
+  const [questionsList, setQuestionsList] = useState<IQuestion[]>([
+    initialQuestion,
+  ]);
+  const answerTypeItems = useMemo(() => getItems(), []);
+
+  function handleSaveForm() {
+    let formToSave = {
+      name: "Form Name",
+      schemaVersion: "v1",
+      fields: questionsList.map((question) => {
+        return {
+          question: question.question,
+          answerType: question.answerType,
+        };
+      }),
+    };
+    console.log("SAAAAVEEEE", formToSave);
```

Comment:
console needed?

Reply:
Didn't want to call the sdk at this stage, it's a pleaceholder till then